### PR TITLE
Added site information for the Otehiwai Observatory in NZ

### PR DESCRIPTION
**Pull request by CheerfulUser**
### Description

This pull request is to address ...

Fixes #<Issue Number>

### CODE REVIEW

File: coordinates/sites.json [hunk 1]
```diff
@@ -924,5 +924,23 @@
...
+   "Otehiwai Observatory": {
+        "elevation": 1029,
+        "name": "Otehiwai Observatory",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": -47.985447,
+        "elevation_unit": "meter",
+        "longitude": 170.464154,
+        "timezone": "NZ",
+        "source": "Wikipedia https://en.wikipedia.org/wiki/Mount_John_University_Observatory",
```
Comment (hamogu) [inline coordinates/sites.json]: Wikipedia is not necessarily the best source. Anyone can edit the data there any time or links to stuff become dead. For example, the wikipedia site has a link to the observatory website, which does not work. Of course, we can verify the information with a look at google maps (observatory domes are typically pretty easy to spot), but if at all possible, we would strongly prefer a source that people can still look up later, e.g. a publication in the astronomical literature. Of course, sometimes that does not exist, but it's worth checking.

(I know that not all observatories in the already list have a good source, but we try to see what we can do at least for new data added.)

**Review by hamogu — COMMENTED** [review]
Thanks for the PR; it's good to have people add more data for this to make it more useful for everyone.

File: coordinates/sites.json (hunk 1 above)
Comment (CheerfulUser) [reply coordinates/sites.json]: Very good point, I've updated it with a reliable source.

**Comment by hamogu** [comment]
Thanks for contributing to the database!

**Pull request merged**